### PR TITLE
Add validation tests for bufferSize in B2T T2B copies on depth stenci…

### PR DESCRIPTION
**Pull request by Jiawei-Shao**
-----

- [ ] New helpers, if any, are documented in `helper_index.md`.
- [ ] Incomplete tests, if any, are marked with TODO or `.unimplemented()`.

**[Review requirement](https://github.com/gpuweb/cts/blob/main/docs/reviews.md) checklist:**

- [ ] WebGPU readability
- [ ] TypeScript readability

### CODE REVIEW

File: src/webgpu/api/validation/encoding/cmds/buffer_texture_copies.spec.ts [hunk 1]
```diff
@@ -44,23 +76,92 @@ g.test('depth_stencil_format,copy_usage_and_aspect')
...
+      t.testCopyTexturetoBuffer({ texture, aspect }, { buffer }, textureSize, success);
+    }
+  });
 
-      const encoder = t.device.createCommandEncoder();
-      encoder.copyTextureToBuffer({ texture, aspect }, { buffer }, textureSize);
+g.test('depth_stencil_format,copy_buffer_size')
+  .desc(
+    `
+  Validate the minimum buffer size for each depth stencil format in copyBufferToTexture
```
[the remark this answers is not in the document]
Comment (kainino0x) [reply src/webgpu/api/validation/encoding/cmds/buffer_texture_copies.spec.ts:91]: +1, it's not necessary to go into a lot of detail, but useful to say what kinds of cases are covered.

File: src/webgpu/api/validation/encoding/cmds/buffer_texture_copies.spec.ts [hunk 2]
```diff
@@ -7,13 +7,45 @@ TODO: plan further
```
[the remark this answers is not in the document]
Comment (Kangz) [reply src/webgpu/api/validation/encoding/cmds/buffer_texture_copies.spec.ts:7]: I'm pretty sure we also need to test this condition that's specific to copyBufferToTexture:

> source.offset is a multiple of the texel block size of destination.texture.[[format]].

But I'm not sure if it should be here or in image_copy/texture-related.

File: src/webgpu/api/validation/encoding/cmds/buffer_texture_copies.spec.ts [hunk 3]
```diff
@@ -44,23 +76,92 @@ g.test('depth_stencil_format,copy_usage_and_aspect')
...
+      format,
+      usage: GPUTextureUsage.COPY_SRC | GPUTextureUsage.COPY_DST,
+    });
+
+    const texelAspectSize = depthStencilFormatAspectSize(format, aspect);
+    assert(texelAspectSize > 0);
+
+    const bytesPerRow = align(texelAspectSize * copySize.width, kBytesPerRowAlignment);
+    const rowsPerImage = copySize.height;
+    const minimumBufferSize =
```
[the remark this answers is not in the document]
Comment (Jiawei-Shao) [reply src/webgpu/api/validation/encoding/cmds/buffer_texture_copies.spec.ts:134]: As @kainino0x  said, in this test I am testing the formats that are not included in SizedTextureFormat (like depth24plus-stencil8) so I cannot use that helper function.

**Comment by Jiawei-Shao** [comment]
@kainino0x @Kangz PTAL, thanks!

**Comment by Kangz** [comment]
Still LGTM

**Pull request merged**